cmd/commands/rs/rs.go

```diff
@@ -79,16 +82,19 @@ func runScript(cmd *commands.Command, args []string) int {
 type customCommand struct {
 	Name    string
 	Command string
+	Args    []string
 }
 
 func (c *customCommand) run() error {
 	beeLogger.Log.Info(colors.GreenBold(fmt.Sprintf("Running '%s'...", c.Name)))
 	var cmd *exec.Cmd
 	switch runtime.GOOS {
 	case "darwin", "linux":
-		cmd = exec.Command("sh", "-c", c.Command)
+		args := append([]string{c.Command}, c.Args...)
+		cmd = exec.Command("bash", "-c", strings.Join(args, " "))
```

Comment:
I think you need to stick with `sh -c`. `sh` is a symbolic link to whatever SHELL the user is using. It can be `bash` or `zsh`...etc